src/Plugin/QueueWorker/BentoEventProcessor.php

```
@@ -0,0 +1,270 @@
+   * such as malformed data or authentication issues. Others indicate temporary
+   * issues that may resolve with retry, such as network timeouts.
+   *
+   * @param \Exception $exception
+   *   The exception that occurred during processing.
+   * @param array $event_data
+   *   The event data that was being processed.
+   *
+   * @return bool
+   *   TRUE if the error should trigger a retry, FALSE if the item should be discarded.
+   */
+  private function shouldRetryError(\Exception $exception, array $event_data): bool {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$event_data'. (Unused Code Rules)

(UnusedFormalParameter)